plugins/system/accessibility/accessibility.php

```diff
@@ -96,7 +96,7 @@ public function onBeforeCompileHead()
 							'units' => 'px',
 						],
 					],
-					'useEmojis' => true,
+					'useEmojis' => $this->params->get('useEmojis') != "false" ? true : false,
```

Comment:
```suggestion
					'useEmojis' => $this->params->get('useEmojis') != 'false' ? true : false,
```
We should use single quotes for strings in PHP whenever possible. The reason why a bit below in code double quotes are used (which might have inspired you to you them here, too) is because they are within a PHP string.

As it's only a code style change, applying this suggestion here will not invalidate previous human test results. I will restore them after this suggestion has been applied.

Reply:
Thank you for the explanation.

If only we had a code linter it wuld save so much time